spec/spec_tests/max_staleness_spec.rb

```diff
@@ -139,6 +139,7 @@
             expect do
               server_selector.select_server(cluster)
             end.to raise_exception(Mongo::Error::NoServerAvailable)
+            expect(spec.suitable_servers).to eq([])
```

Comment:
`spec` is actually an object representing the yaml file that the test is based on, so I think you'll want to use this as the expected value in your tests.

Take a look at line 128 for an example of something similar -- `in_latency_window` is derived from `spec.in_latency_window`. We take the results of server selection and make sure that they match the value specified by the yml file.